<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>931</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>931</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>686</width>
    <height>931</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color:rgb(42, 42, 42);
color:rgb(255, 234, 0);
	font: 63 26pt &quot;URW Gothic L&quot;;
	font-weight:bold;
}
QPushButton:hover {
background-color:rgb(100, 100, 100);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>420</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/gameui/pic/Title.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>750</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>691</width>
      <height>941</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="pic.qrc">:/gameui/pic/Title_BG.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Draw Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>640</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Gothic L</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Manage Cards </string>
    </property>
   </widget>
   <widget class="QWidget" name="widgetDrawCard" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>686</width>
      <height>931</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>686</width>
       <height>931</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgba(0,0,0,0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDrawOk">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>820</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>235</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelStone">
    <property name="geometry">
     <rect>
      <x>78</x>
      <y>230</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(0,0,0,0.7);
color:rgb(255, 234, 0);
padding: 5px;
border-width: 1px;
border-style: solid;
border-radius: 4px;
font-weight:bold;
font-size:15pt;



</string>
    </property>
    <property name="text">
     <string>999</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widgetCardM" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>581</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(0, 0, 0, 200)</string>
    </property>
    <widget class="QGraphicsView" name="graphicsViewCardM">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>511</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(0,0,231,100);
border: none;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 26pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(255, 234, 0);
background:rgb(0,0,0,0);
border:none;
</string>
     </property>
     <property name="text">
      <string>Card Manager</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelShowCard">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>100</width>
       <height>130</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: rgba(255,255,255,200);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="labelLevel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>540</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 15pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(57, 167, 6);
background:rgb(100,100,100,235);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string>LEVEL</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxhp">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>540</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 15pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(255, 58, 61);
background:rgb(100,100,100,235);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string>MaxHP</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMananeed">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>570</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 15pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(255, 85, 255);
background:rgb(100,100,100,235);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string>ManaNeed</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAttack">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>540</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 15pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color: rgb(255, 222, 34);
background:rgb(100,100,100,235);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string>Attack</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAttackrange">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>570</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 15pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(255, 180, 75);
background:rgb(100,100,100,235);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string>AttackRange</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelHelp">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>600</y>
       <width>411</width>
       <height>101</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 15pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(182, 160, 255);
background:rgb(100,100,100,235);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="labelName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>670</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 12pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(255, 255, 255);
background:rgb(100,100,100,150);
border:none;
padding:5px;
</string>
     </property>
     <property name="text">
      <string>Choose</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCloseCardM">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>720</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color:rgb(42, 42, 42);
color:rgb(255, 234, 0);
	font: 63 26pt &quot;URW Gothic L&quot;;
	font-weight:bold;
}
QPushButton:hover {
background-color:rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Back To Menu</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetMsgBox" native="true">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>300</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(0, 0, 0, 200)</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 26pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:rgb(255, 234, 0);
background:rgb(0,0,0,0);
border:none;
</string>
     </property>
     <property name="text">
      <string>Warning</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>391</width>
       <height>141</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">	font: 63 20pt &quot;URW Gothic L&quot;;
	font-weight:bold;
color:white;
background:rgb(0,0,0,0);
border:none;
</string>
     </property>
     <property name="text">
      <string>You don't have enough stones, so you can't draw a card.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCloseMsgBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>391</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color:rgb(42, 42, 42);
color:rgb(255, 234, 0);
	font: 63 26pt &quot;URW Gothic L&quot;;
	font-weight:bold;
}
QPushButton:hover {
background-color:rgb(100, 100, 100);
}</string>
     </property>
     <property name="text">
      <string>Back To Menu</string>
     </property>
    </widget>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>labelStone</zorder>
   <zorder>label_3</zorder>
   <zorder>widgetDrawCard</zorder>
   <zorder>widgetCardM</zorder>
   <zorder>widgetMsgBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
